app/services/subscriptions/create_service.rb

```diff
@@ -128,6 +128,10 @@ def create_subscription
         after_commit { SendWebhookJob.perform_later('subscription.started', new_subscription) }
       end
 
+      if new_subscription.should_sync_crm_subscription?
+        after_commit { Integrations::Aggregator::Subscriptions::Crm::UpdateJob.perform_later(subscription: new_subscription) }
```

Comment:
I noticed that here we should call the create service:

```suggestion
        after_commit { Integrations::Aggregator::Subscriptions::Crm::CreateJob.perform_later(subscription: new_subscription) }
```